### ISSUE TITLE
Reveal: Uncaught TypeError: Cannot read property 'find' of undefined

### ISSUE THREAD

**Issue by phifa** [zurb/foundation-sites#8492]
Go to http://foundation.zurb.com/sites/docs/reveal.html#animations and click on the reveal link: 
*Click me for a modal* 

Watch the Console output: 

````
Uncaught TypeError: Cannot read property 'find' of undefined
```

**Comment by Owlbertz** [comment]
Possible duplicate of #8487?

**Comment by phifa** [comment]
I don't know if it is fixed yet. I guess we will see with the release next week.

**Comment by Owlbertz** [comment]
Are you using the development version or the stable 6.2 release?

**Comment by phifa** [comment]
the one that comes with the zurb template? i guess stable?

**Comment by phifa** [comment]
can i pass a parameter or something in the zurb template to have it use the dev version instead of stable?

**Comment by Owlbertz** [comment]
Well you would have to download the files from the Github Dev branch, i.e. [here](https://github.com/zurb/foundation-sites/archive/develop.zip). Would be great if you could check if this version would solve the issue.
The fix would then be available in the stable version as soon as 6.2.1 will be released.

**Comment by elicohenator** [comment]
hi, i'm also having this issue, 
in my case i've added some MotionUI parameters to the Reveal. for example: 
`<div class="reveal" id="exampleModal1" data-reveal data-animation-in="slide-in-up" data-animation-out="slide-in-down">`
after removing them the error disappeared.
any help would be appriciated :)

**Comment by iammichiel** [comment]
Same behaviour as @elicohenator. 
When removing all the animation classes, everything works fine. Might be motion-ui related then.

**Comment by brettsmason** [comment]
I ran into this issue myself yesterday using the stable 6.2 branch along with motion UI animations on the reveal. I tested the dev branch on the same project as @Owlbertz suggested and it works correctly.

I'm going to close this as the fix will be includes in 6.2.1 (hopefully out this week). If you still have this issue after testing 6.2.1 please reopen this!
